### PR TITLE
Feature/make command configurable

### PR DESCRIPTION
**Pull request by t-b**
I'm not a cmake expert but with this patch I can set the debugging command myself.

### CODE REVIEW

File: cmake/CreateLaunchers.cmake [hunk 1]
```diff
@@ -235,11 +243,11 @@ macro(_launcher_produce_vcproj_user)
 
 		set(USERFILE_CONFIGSECTIONS)
 		foreach(USERFILE_CONFIGNAME ${config_types})
-            set(USERFILE_COMMAND "$<TARGET_FILE:${_targetname}>")
+      set(USERFILE_COMMAND "${COMMAND}")
```
Comment (ukos-git) [inline cmake/CreateLaunchers.cmake:246]: looks like an indentation error

Comment (caseymcc) [reply cmake/CreateLaunchers.cmake:246]: yeah, fix that and Ill merge it, do you use hunter? If so Ill version it and update hunter, if not likely wait.

**Comment by caseymcc** [comment]
also the readme should be updated to include the COMMAND option.

File: cmake/CreateLaunchers.cmake (hunk 1 above)
Comment (t-b) [reply cmake/CreateLaunchers.cmake:246]: @caseymcc I don't use hunter.

The file has many indentation errors. To what should I standardize?

File: README.md [hunk 2]
```diff
@@ -14,21 +14,22 @@ Usage
...
   include(CreateLaunchers)
 
   #Creates bash or shell script to launch the target from commandline
-  # -When using a VS generator it also creates VS .user so that the 
+  # -When using a VS generator it also creates VS .user so that the
   #   target can be launched by the VS debugger with request settings
   # -When using gdb it will create shell scripts that call gdb to start
   #   the target with the requested settings
   create_target_launcher(<targetname>
+    [COMMAND <target command>]
```
Comment (t-b) [inline README.md:27]: @caseymcc The readme already mentions `COMMAND`.

**Comment by caseymcc** [comment]
missed the readme somehow, thanks

**Pull request merged**